buildSrc/src/main/java/io/micronaut/guides/core/GuideUtils.java

```diff
@@ -131,4 +134,14 @@ private static void addAllSafe(Collection target, Collection src) {
             target.addAll(src);
         }
     }
+
+    static boolean shouldSkip(Guide guide, BuildTool buildTool) {
+        if (buildTool == BuildTool.GRADLE) {
```

Comment:
Please, use BuildTool.valuesGradle(). If skipGradleTests is true it should work for either Groovy or Kotlin Gradle DSLs.

Reply:
done